backend/app/migrator/mapper.go

```diff
@@ -0,0 +1,68 @@
+package migrator
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"strings"
+)
+
+// UrlMapper implements Mapper interface
+type UrlMapper struct {
+	rules       map[string]string
+	prefixRules map[string]string
```

Comment:
do we really need separate prefix rules? If rules handles both exact match and some form of regex I would thing they could be combined